### PR TITLE
Test Pie retuns 404 for missing repository

### PR DESCRIPTION
**Pull request by victornoel**
This is for #120.

### CODE REVIEW

**Comment by victornoel** [comment]
@g4s8 gitlint seems to be broken for some old commits...

**Comment by 0crat** [comment]
This pull request [#144](https://github.com/artipie/artipie/issues/144) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c210e0b7-18f7-4097-8918-c85b126950d8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/PieTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * Test for {@link Pie}.
+ * @since 0.2
+ */
+@ExtendWith(VertxExtension.class)
+public final class PieTest {
+    @Test
+    public void unexistingRepoReturnNotFound(final Vertx vertx, @TempDir final Path tmp) {
+        MatcherAssert.assertThat(
+            "Must return 400 HTTP status",
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/PieTest.java:48]: @victornoel You mean _404_?

**Comment by victornoel** [comment]
@carlosmiranda it is fixed, thank you :)

**Comment by carlosmiranda** [comment]
@g4s8 ping

**Comment by victornoel** [comment]
@carlosmiranda I had to resolve some conflicts to rebas eunto master, if you want to verify all is good :)

**Comment by carlosmiranda** [comment]
@victornoel I don't have any further comments. 👍 

@g4s8  - ping

File: src/main/java/com/artipie/Settings.java [hunk 2]
```diff
@@ -49,4 +52,43 @@
...
+         * @param vertx Vertx
+         */
+        public Fake(final Path storage, final Vertx vertx) {
+            this.storage = storage;
+            this.vertx = vertx;
+        }
+
+        @Override
+        public Storage storage() throws IOException {
+            return new FileStorage(this.storage, this.vertx.fileSystem());
```
Comment (g4s8) [inline src/main/java/com/artipie/Settings.java:86]: @victornoel can we use `InMemoryStorage` for testing? I think it has more sense for fake implementation here and it doesn't depend on FS resources

**Comment by victornoel** [comment]
@g4s8 see new commit please

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#144` is not assigned, can't get performer

**Pull request merged**